docs/topics/api/ratings.rst

```diff
@@ -27,6 +27,7 @@ user has already posted a rating for the current version of an add-on.
 .. http:get:: /api/v5/ratings/rating/
 
     :query string addon: The :ref:`add-on <addon-detail>` id, slug, or guid to fetch ratings from. When passed, the ratings shown will always be the latest posted by each user on this particular add-on (which means there should only be one rating per user in the results), unless the ``version`` parameter is also passed.
+    :query string app: Set the :ref:`add-on application <addon-detail-application>` for that query. This won't filter the results, but can affect the URLs being returned. Defaults to ``firefox``.
```

Comment:
I think anything added to rst docs should also be added to swagger at this point. Let's at least not get further behind.